packages/graphql/src/translate/where/property-operations/create-global-node-operation.ts

```diff
@@ -37,7 +37,7 @@ export function createGlobalNodeOperation({
     if (coalesceValue) {
         idProperty = CypherBuilder.coalesce(
             idProperty as CypherBuilder.PropertyRef,
-            new CypherBuilder.Literal(coalesceValue)
+            new CypherBuilder.RawCypher(`${coalesceValue}`) // TODO: move into CypherBuilder.literal
```

Comment:
question (non-blocking): move it as part of this PR or a future PR?

Reply:
Right now coalesceValue is processed outside of the cypher builder, once we moved everything to cypher builder we should be able to get rid of that pre processing